### PR TITLE
Fix typos 

### PR DESCRIPTION
**Pull request by OnkarRuikar**
## Summary
Extending #15179

`left hand` to `left-hand`

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/gamepad/hand/index.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ The **`hand`** read-only property of the {{domxref("Gamepad")}} interface return
 
 A [`GamepadHand`](https://w3c.github.io/gamepad/extensions.html#gamepadhand-enum) enum; possible values are:
 
-- `left` — the left hand.
-- `right` — the right hand.
+- `left` — the left-hand.
+- `right` — the right-hand.
```
Comment (teoli2003) [inline files/en-us/web/api/gamepad/hand/index.md:23]:
```suggestion
- `left` — the left hand.
- `right` — the right hand.
```

File: files/en-us/web/api/webxr_device_api/geometry/index.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ Because of the variety of XR hardware available, coming in a wide variety of for
...
 
-Next, consider the XR controller you have in your left hand. It has the ability to report movement and its orientation, but it doesn't know anything about the position of the headset or, more crucially, its coordinate system. But the controller still needs a way to report its position to your app. Thus, it has its own coordinate system. This is a reference space which is provided to your app when input events occur. This reference space internally knows how to map the coordinates of the controller to the headset's coordinates, so WebXR can translate coordinates back and forth for you.
+Next, consider the XR controller you have in your left-hand. It has the ability to report movement and its orientation, but it doesn't know anything about the position of the headset or, more crucially, its coordinate system. But the controller still needs a way to report its position to your app. Thus, it has its own coordinate system. This is a reference space which is provided to your app when input events occur. This reference space internally knows how to map the coordinates of the controller to the headset's coordinates, so WebXR can translate coordinates back and forth for you.
```
Comment (teoli2003) [inline files/en-us/web/api/webxr_device_api/geometry/index.md:113]:
```suggestion
Next, consider the XR controller you have in your left hand. It has the ability to report movement and its orientation, but it doesn't know anything about the position of the headset or, more crucially, its coordinate system. But the controller still needs a way to report its position to your app. Thus, it has its own coordinate system. This is a reference space which is provided to your app when input events occur. This reference space internally knows how to map the coordinates of the controller to the headset's coordinates, so WebXR can translate coordinates back and forth for you.
```

File: files/en-us/web/api/webxr_device_api/inputs/index.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ With this, you now have the point from which the targeting ray is emitted (`targ
 
 The input source's {{domxref("XRInputSource.gripSpace", "gripSpace")}} property is an `XRSpace` you can use to render objects so they appear to be held in the viewer's hand.
 
-**Figure: The coordinate system for the left hand's grip space.**
+**Figure: The coordinate system for the left-hand's grip space.**
```
Comment (teoli2003) [inline files/en-us/web/api/webxr_device_api/inputs/index.md:101]:
```suggestion
**Figure: The coordinate system for the left hand's grip space.**
```

File: files/en-us/web/api/xrinputsource/gripspace/index.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ Imagine that the controller is shaped like a straight rod, held in the user's fi
 native origin of the grip space is located at the centroid—the center of mass—of the
 user's fist, tracking the position of the user's hand.
 
-**The coordinate system for the left hand's grip space.**
+**The coordinate system for the left-hand's grip space.**
```
Comment (teoli2003) [inline files/en-us/web/api/xrinputsource/gripspace/index.md:35]:
```suggestion
**The coordinate system for the left hand's grip space.**
```

File: files/en-us/web/api/xrinputsource/gripspace/index.md [hunk 5]
```diff
@@ -44,7 +44,7 @@ As shown in the diagram above, the coordinate system is oriented as follows:
 
 - The x-axis is perpendicular to the palm of the user's hand, with the direction
   extending outward from the back of the hand being +X if the controller is in the
-  user's right hand or -X if the controller is in the left hand.
+  user's right hand or -X if the controller is in the left-hand.
```
Comment (teoli2003) [inline files/en-us/web/api/xrinputsource/gripspace/index.md:47]:
```suggestion
  user's right hand or -X if the controller is in the left hand.
```

File: files/en-us/web/api/xrinputsource/handedness/index.md [hunk 6]
```diff
@@ -35,7 +35,7 @@ the user's hands, and if it is, which hand. The value is one of the following:
 - `none`
   - : The input controller is not associated with one of the user's hands.
 - `left`
-  - : The input controller is being held in, worn on, or is attached to the user's left hand.
+  - : The input controller is being held in, worn on, or is attached to the user's left-hand.
```
Comment (teoli2003) [inline files/en-us/web/api/xrinputsource/handedness/index.md:38]:
```suggestion
  - : The input controller is being held in, worn on, or is attached to the user's left hand.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by teoli2003** [comment]
You will need to resolve conflicts here.

**Comment by OnkarRuikar** [comment]
done!

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**